src/reducers/VaccineReducer.js

```diff
@@ -0,0 +1,39 @@
+import { VACCINE_ACTIONS } from '../actions/VaccineActions';
+
+const initialState = () => ({
+  scannedSensorAddresses: [],
+  isScanning: false,
+});
+
+export const VaccineReducer = (state = initialState(), action) => {
+  const { type } = action;
+
+  switch (type) {
+    case VACCINE_ACTIONS.SCAN_START: {
+      return {
+        ...state,
+        isScanning: true,
+      };
+    }
+    case VACCINE_ACTIONS.SCAN_STOP: {
+      return {
+        ...state,
+        isScanning: false,
+      };
+    }
+    case VACCINE_ACTIONS.SENSOR_FOUND: {
+      const { payload } = action;
+      const { macAddress } = payload;
+
+      const { scannedSensorAddresses } = state;
+      scannedSensorAddresses.push(macAddress);
+
+      return {
+        ...state,
+        scannedSensorAddresses,
+      };
+    }
+    default:
+      return state;
+  }
+};
```

Comment:
can add this to the redux store by adding it to `src/reducer/index.js` (If at any point there are realm objects added into the state, we will need to blacklist the state from being persisted in `src/store.js`

Reply:
Added with c239fc74. I did wonder about adding it if it wasn't being used yet. But it's probably more helpful than not to 😅